recipes/conversation_view/feed_pill/feed_item_pill_constants.js

```diff
@@ -1,7 +1,12 @@
 export const FEED_ITEM_PILL_BORDER_COLORS = {
-  default: 'dt-feed-item-pill--border-default',
-  ai: 'dt-feed-item-pill--border-ai',
-  critical: 'dt-feed-item-pill--border-critical',
+  default: 'dt-recipe-feed-item-pill--border-default',
+  ai: 'dt-recipe-feed-item-pill--border-ai',
+  critical: 'dt-recipe-feed-item-pill--border-critical',
+};
+
+export const FEED_ITEM_PILL_DATA_QA = {
+  PILL: 'dt-recipe-feed-item-pill',
+  PILL_ICON: 'dt-recipe-feed-item-pill-icon',
```

Comment:
Don't export constants that are only used for QA, you can just put the content right in place.